tracee-ebpf/tracee/consts_amd64.go

```diff
@@ -798,5 +891,12 @@ const (
 	sys32faccessat2                   int32 = 439
 	sys32process_madvise              int32 = 440
 	sys32epoll_pwait2                 int32 = 441
+	sys32mount_setattr                int32 = 442
+	sys32quotactl_fd                  int32 = 443
+	sys32landlock_create_ruleset      int32 = 444
+	sys32landlock_add_rule            int32 = 445
+	sys32landlock_restrict_self       int32 = 446
+	sys32memfd_secret                 int32 = 447
+	sys32process_mrelease             int32 = 448
```

Comment:
These syscalls are new!
This means we should add them to the "regular" 64bit event ids as well.
On recent kernels (not sure on which kernel version it started), all the architectures get the same syscall numbers for new syscalls. Also see this comment- https://github.com/torvalds/linux/blob/master/arch/x86/entry/syscalls/syscall_64.tbl#L346

Reply:
I will create a new PR for the new syscalls and put it there